docs/how-to-guides/contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
```

Comment:
Discourse actually has an updated version of this which has some nice changes (e.g., linking to our contributing guide): https://github.com/canonical/discourse-k8s-operator/blob/main/docs/how-to-guides/contributing.md